packages/L10n/src/i18n.js

```diff
@@ -9,6 +9,9 @@ export const i18n = locales => {
     fallbackLng: defaultLocale,
     lowerCaseLng: true,
     resources: locales || paprikaLocales,
+    interpolation: {
+      escapeValue: false,
```

Comment:
I don't think we want this @allison-c , why would we want to be false by default? 

```
escapeValue
(default) true
escapes passed in values to avoid XSS injection
```

doesn't `i18next.t('keyEscaped', { myVar: '<img />', interpolation: { escapeValue: false } });` do what you want to?

[interpolation i18next](https://www.i18next.com/translation-function/interpolation)

Reply:
Yes, but there're too many places I need to add `interpolation: { escapeValue: false }` in asset inventory, maybe more in the future. e.g. {{assetName}} details,add new {{assetName}}.... we can also create a helper function for I18n.t with the configuration but I think fix it in paprika side it's easier for developers. 
BTW react will escape strings anyway even if we don't use i18next. As long as we are cautious when using dangerouslyInsert
What do you think if I add another prop on L10n component like `options` and pass it here? So this won't be a breaking change. I put it as a breaking change because some consumer app might already did something to address this issue?

Reply:
But is it necessary, @nahumzs ?

This code:
```
export default function Greeting() {
  const i18n = useI18n();
  return (
    <p>Custom locale greeting: {i18n.t("customlocalegreeting", { name: "<script>alert('what up');</script>" })}</p>
  );
}
```
outputs this:
![Screen Shot 2020-11-24 at 10 56 06 AM](https://user-images.githubusercontent.com/23224777/100139049-c0f68500-2e43-11eb-803e-40e95642dd57.png)

Reply:
@jamiek-acl suggested an easier way to solve the issue by adding `{{-name}}` I'll close the PR 😅

Reply:
@jamiek-acl because by default i18n escape it and react also do it. is a double layer of security as Allison commented. 

I don't think we should disable it by default or even allowed developers to disabled it globally even in their pillars. 


So this is my suggestion, expose a new method in i18n to ease the process for using it, like: 

```
// i18n paprika file
const dangerouslyT = (key, t, options = {})=> {
  return t(key, {interpolation: { escapeValue: false }, ...moreProps})
}
export dangerouslySetText

// your app

import {dangerouslyT, i18n} from "@paprika/l10n"

dangerouslyT("mykey", i18n.t)

```

Reply:
even in this line https://github.com/acl-services/paprika/blob/master/packages/L10n/src/i18n.js#L18

you could do:

```js
export const i18n = locales => {
  const i18nConfig = {
    lng: defaultLocale,
    fallbackLng: defaultLocale,
    lowerCaseLng: true,
    resources: locales || paprikaLocales,
  };

  const newInstance = i18next.createInstance(i18nConfig, err => {
    if (err) return console.log("something went wrong loading i18next", err);
    return null;
  });

  newInstance.dangerously_t = (key, moreProps = {}) => {
     const {t} = newInstance
     return t(key, {interpolation: { escapeValue: false }, ...moreProps})
  }

  return newInstance;
};

```

Reply:
yeah ok, but we should disable escaping at any level just in specific scenarios.

Reply:
> is it a double layer of security

![boxer](https://user-images.githubusercontent.com/23224777/100141215-e933b300-2e46-11eb-9683-f82b3ce2d2cb.png)